requirements/base.in

```diff
@@ -7,5 +7,4 @@ newrelic
 path.py
 requests
 six
-
--e git+https://github.com/edx/codejail.git@4127fc4bd5775cc72aee8d7f0a70e31405e22439#egg=codejail
+edx-codejail
```

Comment:
Just move this to the second position in the list to preserve alphabetic order

Reply:
Done